code/__defines/mobs.dm

```diff
@@ -148,7 +148,7 @@
 #define ANIMAL_SPAWN_DELAY 5 MINUTES
 #define DRONE_SPAWN_DELAY  10 MINUTES
 
-#define CRYOPOD_SPAWN_BONUS	20 MINUTES //Going to sleep in a cryopod takes this much off your respawn time
+#define CRYOPOD_SPAWN_BONUS	20//Going to sleep in a cryopod takes this much off your respawn time in minutes (X = X MINUTES). MINUTES should be placed after the define when used.
```

Comment:
https://github.com/discordia-space/CEV-Eris/blob/75f5eb63659bfec0eea783b53d8df1fef4907d13/code/__defines/mobs.dm#L148-L151
Looking not much consistent